java/arcs/core/data/testdata/BUILD

```diff
@@ -30,3 +31,18 @@ arcs_kt_gen(
     name = "ingest_process_retrieve_gen",
     srcs = ["IngestProcessRetrieve.arcs"],
 )
+
+arcs_manifest(
+    name = "shared_schemas",
+    srcs = ["Schemas.arcs"],
+)
+
+arcs_kt_gen(
+    name = "etl_gen",
```

Comment:
What is "etl"?

Reply:
Sorry -- Extract, Transform, Load.

Reply:
Can you rename this then, that wasn't readily apparent.